scripts/datenlord-monitor-deploy.sh

```diff
@@ -4,29 +4,46 @@ set -o errexit
 set -o nounset
 set -o xtrace
 
+if [ $# -le 0 ]
+then
+    cat <<EOF
+Usage: $0 [deploy|undeploy|helm]
+
+deploy      Deploy datenlord monitor by yaml file.
+helm        Deploy datenlord monitor by helm.
+undeploy    Undeploy all datenlord monitor.
```

Comment:
make this as a function as print_usage()

Reply:
done